third_party/ruby/build-ruby.bzl

```diff
@@ -91,14 +93,19 @@ run_cmd rm -f configure.bak
 # files that includes `-fvisibility=hidden`. To override it, our flag needs to
 # come after, so we inject a flag right before the `-o` option that comes near
 # the end of the command via OUTFLAG.
-OUTFLAG="-fvisibility=default -o" \
-CC="{cc}" \
-CFLAGS="{copts}" \
-CXXFLAGS="{copts}" \
-CPPFLAGS="{sysroot_flag} ${{inc_path[*]:-}} {cppopts}" \
-LDFLAGS="{sysroot_flag} ${{lib_path[*]:-}} {linkopts}" \
+
+# Export these env vars as they will be needed when configuring libyaml which
+# is called with `./configure`
+export OUTFLAG="-fvisibility=default -o"
```

Comment:
If I didn't export these separately then 3.2 wouldn't compile on macos. When compiling libyaml it would fail because linker flags weren't passed in.

Reply:
This seems fine, and more in line with how these variables are meant to be used.